app/src/main/java/io/pslab/activity/MainActivity.java

```diff
@@ -187,7 +187,7 @@ private Fragment getHomeFragment() throws IOException {
                 return HomeFragment.newInstance(ScienceLabCommon.scienceLab.isConnected(), ScienceLabCommon.scienceLab.isDeviceFound());
             case 5:
                 return AboutUsFragment.newInstance();
-            case 7:
+            case 8:
```

Comment:
why you change this case 7 to case 8 ?

Reply:
In navigationdrawer , Rate app is in 7th position(count begins from 0) , After adding that in strings.xml new position of faq is 8.